playwright-e2e/utils/faker-utils.ts

```diff
@@ -33,3 +33,54 @@ export const setPatientParticipantGuid = async (page: Page) => {
   const urlArray = participantResponse.split('/');
   user.patient.participantGuid = urlArray[6];
 };
+
+/**
+ * Set the user guid with a given parameter
+ * @param guid The guid of a study participant
+ */
+export const saveParticipantGuid = (guid: string) => {
+  user.patient.participantGuid = guid;
+};
+
+/**
+ * Takes a date and formats it into MM/DD/YYYY
+ * @param date The date to format into a string that's MM/DD/YYYY
+ * @returns Re-formatted date
+ */
+export const getDateInMonthDayYearFormat = (date: Date): string => {
```

Comment:
I think [getDate() function](https://github.com/broadinstitute/ddp-angular/blob/0b2ba7ebd319f2861ddfa20f5d038772b17be9d6/playwright-e2e/utils/date-utils.ts#L14) is same.

Reply:
oh - honestly didn't know about that one - nice and neat - plus learned about `Intl.DateTimeFormat` - adding changes